### ISSUE TITLE
Error thrown when holding down Shift and Super

### ISSUE THREAD

**Issue by poach3r** [jwharm/java-gi#155]
My program is throwing an error whenever the user holds down Super and Shift at the same time:
```
java.lang.IllegalStateException: Unexpected value: 64
	at org.gnome.gdk.ModifierType.of(ModifierType.java:149)
	at io.github.jwharm.javagi.interop.Interop.intToEnumSet(Interop.java:1088)
	at org.gnome.gtk.EventControllerKey$KeyPressedCallback.upcall(EventControllerKey.java:396)
	at org.gnome.gio.Application.run(Application.java:1087)
	at org.poach3r.MainKt.main(Main.kt:4)
```

**Comment by jwharm** [comment]
Thanks for the bug report.

Java-GI throws an exception when an undefined flag is set. That turns out to be a bit too strict. I will change it, so that unexpected values are logged and ignored.

However, this is strictly incorrect in case of GdkModifierType, according to its [documentation](https://docs.gtk.org/gdk4/flags.ModifierType.html):

_Note that GDK may add internal values to events which include values outside of this enumeration. Your code should preserve and ignore them. You can use GDK_MODIFIER_MASK to remove all private values._

So Java-GI can't just silently ignore the internal flags, because we're supposed to _preserve_ them. To do that, I will add all possible "internal" flags to GdkModifierType, labeled `INTERNAL_...`, and hope that this horrible sort of API design doesn't exist elsewhere.

**Comment by jwharm** [comment]
I published release 0.11.1-SNAPSHOT with the fix. Can you please confirm if it works for you?

For snapshot releases, add `https://s01.oss.sonatype.org/content/repositories/snapshots` to the repositories in your build script.

**Comment by poach3r** [comment]
This worked for me, thanks!
